src/chrome.manifest

```diff
@@ -1,39 +1,53 @@
+locale https-everywhere ca chrome/locale/ca/
+locale https-everywhere cs chrome/locale/cs/
+locale https-everywhere da chrome/locale/da/
 locale https-everywhere de chrome/locale/de/
-locale https-everywhere ru chrome/locale/ru/
-locale https-everywhere zh-CN chrome/locale/zh-CN/
-locale https-everywhere zh-TW chrome/locale/zh-TW/
+locale https-everywhere el chrome/locale/el/
+locale https-everywhere en chrome/locale/en/
+locale https-everywhere en-GB chrome/locale/en_GB/
 locale https-everywhere es chrome/locale/es/
-locale https-everywhere nl chrome/locale/nl/
-locale https-everywhere ar chrome/locale/ar/
-locale https-everywhere fa chrome/locale/fa/
+locale https-everywhere et chrome/locale/et/
 locale https-everywhere eu chrome/locale/eu/
+locale https-everywhere fa chrome/locale/fa/
+locale https-everywhere fi chrome/locale/fi/
+locale https-everywhere fo chrome/locale/fo/
 locale https-everywhere fr chrome/locale/fr/
-locale https-everywhere pl chrome/locale/pl/
-locale https-everywhere sk chrome/locale/sk/
-locale https-everywhere ms chrome/locale/ms/
-locale https-everywhere da chrome/locale/da/
-locale https-everywhere el chrome/locale/el/
+locale https-everywhere fr-CA chrome/locale/fr_CA/
+locale https-everywhere he chrome/locale/he/
+locale https-everywhere hr-HR chrome/locale/hr_HR/
 locale https-everywhere hu chrome/locale/hu/
 locale https-everywhere it chrome/locale/it/
-locale https-everywhere tr chrome/locale/tr/
+locale https-everywhere ja chrome/locale/ja/
+locale https-everywhere km chrome/locale/km/
 locale https-everywhere ko chrome/locale/ko/
-locale https-everywhere cs chrome/locale/cs/
-locale https-everywhere hr chrome/locale/hr/
-locale https-everywhere he chrome/locale/he/
-locale https-everywhere fi chrome/locale/fi/
+locale https-everywhere lt chrome/locale/lt/
+locale https-everywhere lv chrome/locale/lv/
+locale https-everywhere ms-MY chrome/locale/ms_MY/
 locale https-everywhere nb chrome/locale/nb/
+locale https-everywhere nl chrome/locale/nl/
+locale https-everywhere pl chrome/locale/pl/
+locale https-everywhere pt chrome/locale/pt/
+locale https-everywhere pt-BR chrome/locale/pt_BR/
+locale https-everywhere ro chrome/locale/ro/
+locale https-everywhere ru chrome/locale/ru/
+locale https-everywhere si-LK chrome/locale/si_LK/
+locale https-everywhere sk chrome/locale/sk/
+locale https-everywhere sk-SK chrome/locale/sk_SK/
 locale https-everywhere sl chrome/locale/sl/
-locale https-everywhere bg chrome/locale/bg/
-locale https-everywhere ja chrome/locale/ja/
-locale https-everywhere si chrome/locale/si/
+locale https-everywhere sl-SI chrome/locale/sl_SI/
+locale https-everywhere sq chrome/locale/sq/
+locale https-everywhere sr chrome/locale/sr/
+locale https-everywhere sv chrome/locale/sv/
+locale https-everywhere th chrome/locale/th/
+locale https-everywhere tr chrome/locale/tr/
+locale https-everywhere uk chrome/locale/uk/
+locale https-everywhere zh-CN chrome/locale/zh-CN/
```

Comment:
Actually that's the real path on disk. I'm not sure why some of the directories from Transifex have dashes vs spaces, but *shrug*.